src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -688,7 +688,10 @@ ApiGlobe.prototype.pan = function pan(pVector) {
  * @return     {number}  The zoom level.
  */
 ApiGlobe.prototype.getZoomLevel = function getZoomLevel() {
-    return this.scene.getMap().getZoomLevel();
+    const theoricalZoom = this.scene.getMap().computeTheoricalZoom(this.scene.currentCamera(), this.getRange());
+    // const idNode = this.scene.gfxEngine.screenCoordsToNodeId();
+    // const realZoom = this.scene.getMap().getZoomLevelFromIdnode(idNode);
+    return theoricalZoom;
```

Comment:
The zoom level returned is calculated with best tile's level (in WGS84) whose projection (on screen) doesn't exceed 256 pixels. There is always tiles so no problem if there is not WMTS.

These tiles are geometric tiles.

The scheme tile geometric is WGS84, so a one WMTS image tile in WGS84 is applied to a geometric of the same level.  

On the other hand, several PM tiles of level (N+1) are applied to a geomertic tile of level N.

The scale  is also correct.